mcxisp/mcxisp.c

```diff
@@ -0,0 +1,608 @@
+/*
+ * Phoenix-RTOS
+ *
+ * mcxisp - tool for MCX N94x series ISP
+ *
+ * Copyright 2024 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <stddef.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <termios.h>
+#include <sys/stat.h>
+
+/* clang-format off*/
+#define TTY_DEBUG(fmt, ...) if (0) printf(fmt, ##__VA_ARGS__);
```

Comment:
I was too lazy to add MISRA braces :skull: 
```suggestion
#define TTY_DEBUG(fmt, ...) if (0) { printf(fmt, ##__VA_ARGS__); }
```